### PR TITLE
Make language tool multi-lingual #383

### PR DESCRIPTION
**Pull request by aethelwulffe**
Check for what the actual collation is instead of just assuming it.
This could use more refinements, and need to make sure mysqli actually
works, which it probably doesn't.  This is a WIP commit.

### CODE REVIEW

File: interface/language/lang_definition.php [hunk 1]
```diff
@@ -57,9 +57,10 @@
 <?php
 
 // set up the mysql collation string to ensure case is sensitive (or insensitive) in the mysql queries
+
 if (!$disable_utf8_flag) {
   $case_sensitive_collation = "COLLATE utf8_bin";
-  $case_insensitive_collation = "COLLATE utf8_general_ci";
+  $case_insensitive_collation = $mysqli->character_set_name();
```
Comment (tmccormi) [inline interface/language/lang_definition.php]: This doesn't seem right.  why would you hard code one and lookup the other...  seems like just reading the database setting and using that across the board would be the best approach

Comment (aethelwulffe) [inline interface/language/lang_definition.php]: You are right.  Should that not be something that the controller grabs for session data, or am I still thinking too small and too late?  Would that not be better than checking it here every time?
I imaging that the libreehr user has permissions to ask the database what the collation is.

**Comment by tmccormi** [comment]
I would think so, we should just be able to get that info for the database.   The tricky widget is when individual tables have different character sets and collations (which I have seen in upgrades).  but that might not be a problem for LEHR going forward.

**Comment by aethelwulffe** [comment]
We add that to the migration checklist "give all tables the same collation"

**Comment by aethelwulffe** [comment]
Dropping this commit in 24 hours.

**Pull request closed without merging**